### PR TITLE
[WIP][5G-MVC:pipelined] SMF-UPF Merge testing

### PR DESCRIPTION
**Pull request by panyogesh**
1. Merged with latest #3268 proto
2. Node Manager changes
3. Session Manager changes
4. QER changes

Signed-off-by: YOGESH PANDEY <yogesh@wavelabs.ai>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by panyogesh** [comment]
Commands for verification :
1. Add the PDR with QER Rule
----------------------------------------------------------------------
python pipelined_cli.py  ng_services set_smf_session --subscriber_id=IMSI001222333 --pdr_state=ADD --in_teid=100 --out_teid=200 --ue_ip_addr=192.168.128.11 --gnb_ip_addr=192.168.60.141 --add_rule_id=rule1 --ipv4_dst="192.168.0.0/16" --allow="YES" --priority=10

2. Update PDR Rule (remove rule1 and add rule2)
----------------------------------------------------------------------
python pipelined_cli.py  ng_services set_smf_session --subscriber_id=IMSI001222333 --pdr_state=ADD --in_teid=100 --out_teid=200 --ue_ip_addr=192.168.128.11 --gnb_ip_addr=192.168.60.141 --del_rule_id=rule1 --add_rule_id=rule2 --ipv4_dst="192.168.129.0/24" --allow="YES" --priority=10

3. Delete the PDR
-----------------------------------------------------------------------
python pipelined_cli.py  ng_services set_smf_session --subscriber_id=IMSI001222333 --pdr_state=DEL --in_teid=100 --out_teid=200 --ue_ip_addr=192.168.128.11 --gnb_ip_addr=192.168.60.141 --del_rule_id=rule2

**Comment by panyogesh** [comment]
Fixed minor issue in PDR Delete.

**Comment by panyogesh** [comment]
[Traffic-Test-Setup-UPF.txt](https://github.com/magma/magma/files/5506014/Traffic-Test-Setup-UPF.txt)

**Comment by panyogesh** [comment]
[SMF-Stub-CLI.txt](https://github.com/magma/magma/files/5522388/SMF-Stub-CLI.txt)
[Set-SMF-Message-Dump-Nov-11.txt](https://github.com/magma/magma/files/5522389/Set-SMF-Message-Dump-Nov-11.txt)
[Uplink-Traffic-Verification-Logs-Nov-11.txt](https://github.com/magma/magma/files/5522392/Uplink-Traffic-Verification-Logs-Nov-11.txt)
[Downlink-Traffic-Verification-Logs-Nov-11.txt](https://github.com/magma/magma/files/5522394/Downlink-Traffic-Verification-Logs-Nov-11.txt)

**Comment by panyogesh** [comment]
[sessiond.gz](https://github.com/magma/magma/files/5523005/sessiond.gz)

File: lte/gateway/c/session_manager/SetMessageManagerHandler.cpp [hunk 1]
```diff
@@ -72,50 +72,95 @@ void SetMessageManagerHandler::SetAmfSessionContext(
...
-            (cfg.common_context.sm_session_state() == CREATING_0)) {
-          /* it's new UE establisment request and need to create the session
-           * context
-           */
-          MLOG(MDEBUG)
-              << "AMF request type INITIAL_REQUEST and session state CREATING";
-          /* Read the SessionMap from global session_store,
-           * if it is not found, it will be added w.r.t imsi
-           */
+
```
Comment (uri200) [inline lte/gateway/c/session_manager/SetMessageManagerHandler.cpp:75]: isn't this function changed in this pr? https://github.com/magma/magma/pull/3366/files

Maybe you need to rebase

**Comment by panyogesh** [comment]
Only for demo purpose.

**Pull request closed without merging**